cdk/DevPipeline.py

```diff
@@ -0,0 +1,56 @@
+from aws_cdk import (
+    core
+)
+from aws_cdk.pipelines import CodePipeline, CodePipelineSource, ShellStep, ManualApprovalStep
+
+from makerspace import MakerspaceStage
+from accounts_config import accounts
+
+class DevPipeline(core.Stack):
+    def __init__(self, app: core.App, id: str, *,
+                 env: core.Environment) -> None:
+        super().__init__(app, id, env=env)
+
+        deploy_cdk_shell_step = ShellStep("Synth",
+            input=CodePipelineSource.connection("Mjtlittle/unified-makerspace", "pipeline-development",
+                connection_arn="arn:aws:codestar-connections:us-east-1:577283310524:connection/2e5ad00f-c205-4b57-a752-6aa33d55236c"
+            ),
+            commands=[
```

Comment:
I could see value in trying to add multiple build steps and decoupling the logic for each of these 3.